manifests/action_deploy_artifacts.pp

```diff
@@ -4,12 +4,20 @@
 
 class aem_curator::action_deploy_artifacts (
   $tmp_dir,
+  $apache_conf_dir,
+  $author_port,
+  $author_secure,
+  $author_host,
+  $dispatcher_conf_dir,
+  $docroot_dir,
+  $httpd_conf_dir,
```

Comment:
Also remove/replace references to `httpd_conf_dir` with `apache_conf_dir` to avoid duplication.
As a note, an example value of this dir is `/var/www/html`, this is where httpd.conf lives.

Reply:
Replaced httpd_conf_dir with apache_conf_dir in all files.